### DISCUSSION TITLE
How to use secondary storage backend in network share?

### DISCUSSION THREAD

**Discussion by dtnguyen3011** [ccache/ccache#1114] [Questions & answers] [answered]
Hi, when i try to write cache to secondary storage in network share (ex: file:/abc123.example.com/ccache). It's can't write to this folder in network share. But It's create folder abc123.example.com/ccache in local repo folder.

I used OS - Windows

Pls help me this case.

Thank you so much.

**Comment by jrosdahl** [comment, marked as answer]
Having the cache on a network share is currently not supported by ccache. (If it would by supported, the correct syntax would be file://abc123.example.com/ccache.)

**Reply by dtnguyen3011** [reply]
hi @jrosdahl,
i had try this syntax , but cache don't understand and return invalid path

**Reply by jrosdahl** [reply]
Yes. As I mentioned, ccache currently doesn't support that.

**Reply by dtnguyen3011** [reply]
hm, i got it

Thanks for your support.

**Reply by dtnguyen3011** [reply]
hi @jrosdahl ,
Do you know why primary storage can write to network share, but secondary can't write?

**Reply by jrosdahl** [reply]
Short answer: Because nobody has implemented support for it.

Can you give me an example of how you configure the primary storage location?

**Reply by dtnguyen3011** [reply]
i used it: `CCACHE_DIR = \\abc123.example.com\ccache`

It's worked

**Reply by jrosdahl** [reply]
Thanks. I'll implement support for this.

**Reply by dtnguyen3011** [reply]
thank you so much :D

**Reply by jrosdahl** [reply]
Done: cc5c78374e320bc7807f04391b70e43deaa54c94

**Reply by dtnguyen3011** [reply]
in this link: https://github.com/ccache/ccache/releases/tag/v4.6.1
Which package can I use the new feature you updated?

**Reply by jrosdahl** [reply]
You can't yet. It will be part of version 4.7 whenever that is released.

**Reply by dtnguyen3011** [reply]
so great!

**Reply by dtnguyen3011** [reply]
hi Mr @jrosdahl  do you know when version 4.7 will be release?

**Reply by jrosdahl** [reply]
No.
